src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAuthenticationManager.java

```diff
@@ -1209,14 +1239,7 @@ public Cookie createAuthLogoutCookie() {
     }
 
     public NewCookie createAuthLogoutNewCookie() {
-        NewCookie logoutCookie = new NewCookie.Builder(SEGUE_AUTH_COOKIE)
-                .value("")
-                .path("/")
-                .maxAge(0)
-                .httpOnly(true)
-                .secure(setSecureCookies)
-                .comment(SAME_SITE_LAX_COMMENT)
-                .build();
+        NewCookie logoutCookie = new NewCookie.Builder(SEGUE_AUTH_COOKIE).value("").path("/").maxAge(0).httpOnly(true).secure(setSecureCookies).comment(SAME_SITE_LAX_COMMENT).build();
```

Comment:
this was clearer before I think?

Reply:
Hmm... must have been caught in a code format without me noticing. Ah, just after merging in the latest changes from main, I think. Agreed on the clarity, I'll put that one back...